server/src/test/java/com/broadcom/lsp/domain/CopybookStorableProvider.java

```diff
@@ -29,10 +29,10 @@
   private static final String COPYBOOK_CONTENT = "000000 IDENTIFICATION DIVISION.";
 
   @Singleton
-  private static CopybookStorable STORABLE_INSTANCE =
-      new CopybookStorable(COPYBOOK_NAME, COPYBOOK_URI, COPYBOOK_CONTENT);
+  private static CopybookModel STORABLE_INSTANCE =
+      new CopybookModel(COPYBOOK_NAME, COPYBOOK_URI, COPYBOOK_CONTENT);
 
-  protected CopybookStorable getDummyStorable() {
+  protected CopybookModel getDummyStorable() {
```

Comment:
I think we may get rid of the "dummy stuff" loaded in this provider, I saw that in some test we create an instance of ```CopybookModel``` directly though its constuctor